website/pizzas/static/pizzas/js/admin.js

```diff
@@ -109,4 +90,47 @@ django.jQuery(function() {
             }
         });
     });
+    $.tablesorter.addParser({
+        id: 'payment',
+        is: function(s) {
+            return false;
+        },
+        format: function(s, t, node) {
+            const val = node.firstElementChild.value;
+            if (val === 'no_payment') {
+                return 'z';
+            }
+            return val;
+        },
+        type: "text",
+    });
+
+    $.tablesorter.addParser({
+        id: "sortval",
+        is: function(s) {
+            return false;
+        },
+        format: function(s, t, node) {
+            return $(node).data("sortval");
+        },
+        type: "text"
+    });
+
+    $("table").tablesorter({
+        sortList: [[1,0]],
```

Comment:
Why the sortList?

Reply:
I just copied it from what was there earlier. It states that the table should be sorted on the second column (last name) in ascending order

Reply:
I know what it does haha, but it is added in this PR. It is in the events admin.js but not in the original one for the pizza admin.

Reply:
Deliberate choice not to change this back?

Reply:
I figured it would be better not to change behaviour (sorting on first name is still possible by clicking it)